src/plugins/intel_cpu/src/emitters/plugin/x64/jit_eltwise_emitters.cpp

```diff
@@ -293,15 +294,15 @@ void jit_multiply_emitter::emit_isa(const std::vector<size_t>& in_vec_idxs,
 }
 
 std::set<std::vector<element::Type>> jit_multiply_emitter::get_supported_precisions(
-    const std::shared_ptr<ov::Node>& node) {
+    [[maybe_unused]] const std::shared_ptr<ov::Node>& node) {
     return {{element::f32, element::f32}, {element::i32, element::i32}};
 }
 
 /// DIVIDE ///
 jit_divide_emitter::jit_divide_emitter(x64::jit_generator* host,
                                        x64::cpu_isa_t host_isa,
                                        const std::shared_ptr<ov::Node>& node,
-                                       ov::element::Type exec_prc)
+                                       ov::element::Type /*exec_prc*/)
```

Comment:
What is the reason behind commenting out this parameter, while in other contexts such variables are marked [[maybe_unused]] ?